docs/tutorials/basic/ndarray_indexing.md

```diff
@@ -0,0 +1,377 @@
+
+# NDArray Indexing - Array indexing features
```

Comment:
why not merge this into the ndarray tutorial?

Reply:
There wasn't time to integrate it, and the older tutorial has a lot of separate issues that need to be addressed.